crates/ruff/resources/test/fixtures/flake8_type_checking/TCH002.py

```diff
@@ -44,7 +44,7 @@ def f():
 
 
 def f():
-    import pandas as pd
+    import pandas as pd  # TCH002
```

Comment:
This _should_ error because it's _not_ required at runtime, despite being on the RHS.